### PR TITLE
Improve output formatting for spelling suggestions

### PR DESCRIPTION
**Pull request by c0nst4ntin**
### What:

- [ ] Bug Fix
- [X] New Feature

### Description:
This pull request aims to add an improved styling to the CLI output:

peck-cli-output

**Fun Fact:** Not my first OS contribution, but my first time using Termwind! Will definitely use it in future projects!

### CODE REVIEW

**Comment by xiCO2k** [comment]
Awesome look!! LETS GOOOOOOOOOOO

File: src/Console/Commands/DefaultCommand.php [hunk 1]
```diff
@@ -54,10 +54,20 @@ protected function renderIssue(OutputInterface $output, Issue $issue, string $cu
...
+        $lineInfo = ($issue->line !== 0) ? " on line <strong>{$issue->line}</strong>" : '';
+        $suggestions = implode(', ', $issue->misspelling->suggestions);
 
         render(<<<HTML
-            <div class="text-red-700 pl-2">
-                <span>{$issue->misspelling->word} in {$file}.</span>
+            <div class="mx-2 mb-1">
+                <div class="space-x-1">
+                    <span class="bg-red text-white px-1 font-bold">ISSUE</span>
+                    <span>Misspelling detected in <strong>{$file}</strong>{$lineInfo}: '<strong>{$issue->misspelling->word}</strong>'</span>
```
Comment (xiCO2k) [inline src/Console/Commands/DefaultCommand.php:64]: I would add an `<a>` tag to open the file here.

**Review by xiCO2k — COMMENTED** [review]
All the rest looks awesome!!

File: src/Console/Commands/DefaultCommand.php (hunk 1 above)
Comment (c0nst4ntin) [reply src/Console/Commands/DefaultCommand.php]: @xiCO2k Great recommendation, I added this feature immediately!

**Comment by c0nst4ntin** [comment]
If you want to expand on this (e.g., maybe by adding an empty state if everything was spelt correctly or a simple header, etc.), just let me know. I'd be happy to also implement this as well. 

For now I wanted to keep it simple, as I don't want to unnecessarily blow up the CLI output.

**Comment by razvaniacob** [comment]
@c0nst4ntin for some reason when I click, even cmd+click on the links to the files outputted in the issues list, it doesn't open that file in my IDE. I tried to solve it but couldn't get it to work with <a href> tag. THe only way it's working for me if to remove the entire <a> tag and output the full path of the file.

Any idea why and how to solve it?

**Pull request merged**